### ISSUE TITLE
Button: poor look if "label" property is omitted

### ISSUE THREAD

**Issue by masmau** [melfore/mosaic#63]
```js
        <Button
          onClick={()=>{}}
          label="my button"
        >
```
![image](https://user-images.githubusercontent.com/16205876/79049723-985bab00-7c25-11ea-8700-1bdd9162d614.png)

```js
        <Button
          onClick={()=>{}}
        >
```
![image](https://user-images.githubusercontent.com/16205876/79049746-b6291000-7c25-11ea-958b-aaab2d033254.png)

**Comment by masmau** [comment]
Since label should be mandatory, this issue can be solved together with #71

**Comment by luciobordonaro** [comment]
Yes, I totally agree, let's close it as duplicate and use #71 as reference.
